Refactor `chrome_settings_overrides` page to use `<dl>` syntax (#42580)
mdn/content: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md

    The replacement is given as a URL. The URL may:
    - point to a file bundled with the extension, in which case it is
      given as a URL relative to the manifest.json file
    - be a remote URL, such as "https://developer.mozilla.org/".

    If two or more extensions both set this value, then the setting from
    the most recently installed one will take precedence.

    To override new tabs, use [chrome_url_overrides](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/chrome_url_overrides) instead.

    This is a [localizable property](/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#internationalizing_manifest.json).

- `search_provider`
  - : Defines a search provider to add to the browser.

    The search provider has a name and a primary search URL. Alternative
    URLs may be provided, including URLs for more specialized searches
    like image search. In the URL you supply, use
    `{searchTerms}` to interpolate the search term into the
    URL, like:
    `https://www.discogs.com/search/?q={searchTerms}`. You can
    also provide POST parameters to be sent along with the search.

>     The search provider will be presented to the user alongside the

Reviewer: ```suggestion
    The search provider is presented to the user alongside the
```
